src/config/security.ts

```diff
@@ -59,12 +70,15 @@ export function getTokenDuration(): number {
     return parsedDuration;
 }
 
+/**
+ * @internal
+ */
 export function issueToken(userId: string|null = null): Promise<Token> {
     const token = Token.issue(getAppId(), userId)
         .withDuration(getTokenDuration());
 
     if (isUserTokenAuthenticationEnabled()) {
-        return token.withTokenId(uuid())
+        return token.withTokenId(crypto.randomUUID())
```

Comment:
This will break on environments that don't have a `crypto` global. `uuid` has a problem with their fallback, but we'd be replacing one problem for another.

Next.JS up to 13 will enforce it locally and it can be deployed to runtimes without the global even tho Vercel's Edge runtime has support nowadays.

Firecracker from AWS doesn't and neither does Cloudflare without requiring enabling some compatibility flags (which users would have to guess, the error is very non-descriptive)

Reply:
As we discussed internally, signing the token would already require web crypto, so this change is fine.

Users using the library only for anonymous user now have the requirement of web crypto in the environment to generate the `clientId` cookie, but since that comes from the same middleware those users would always have a token as well.